### PR TITLE
Fixes #15702: typo in path name

### PR DESCRIPTION
**Pull request by beav**
The ISS docs mention `/var/www/pub/` as the exposed path which is
incorrect, it should be `/var/www/html/pub/`.

### CODE REVIEW

**Comment by beav** [comment]
Thanks @jlsherrill for catching this

File: docs/user_guide/disconnected/index.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ single-layer DVD.
 
 Importing (step 3) can be done in one of two ways. The first way is to make the
 export available via HTTP to the importing Satellite instance. Simply put the
-export in `/var/www/pub/export`. After that, edit your CDN location from the
+export in `/var/www/html/pub/export`. After that, edit your CDN location from the
```
Comment (ehelms) [inline docs/user_guide/disconnected/index.md]: This is where Pulp exports content to as well with our current configuration? Would there ever be a clash?

Comment (beav) [inline docs/user_guide/disconnected/index.md]: This is a good point, the docs may need a more general going-over to account for usability tweaks that we added. I'll do that now and update this PR.

File: docs/user_guide/disconnected/index.md [hunk 2]
```diff
@@ -55,10 +54,14 @@ single-layer DVD.
...
-manifest import page to point to "http://localhost/export/path/to/export" and
-the Red Hat Repos page will then work as expected, using your exported data.
+export available via HTTP to the importing Katello instance. Simply put the
+export in `/var/www/html/pub/export`, either via copy or symlink. After that,
+edit your CDN location from the manifest import page to point to
+"http://<hostname>/export/path/to/export" and the Red Hat Repos page will then
+work as expected, using your exported data. Please be sure to use 'http' and
+not 'https' when altering the CDN url. Katello by default only supports the CA
+certificate for `cdn.redhat.com`. This is a known limitation that will be
+addressed in a future version.
```
Comment (thomasmckay) [inline docs/user_guide/disconnected/index.md]: Is there an issue to link here?

**Comment by beav** [comment]
updated to add link to redmine issue

**Comment by ehelms** [comment]
ACK - thanks @beav

**Pull request merged**